spec/lib/remote_translations/caller_spec.rb

```diff
@@ -206,4 +207,22 @@
       end
     end
   end
+
+  describe "#field values" do
+    let!(:proposal)          { create(:proposal, description: "&Sigma; with sample text") }
+    let(:remote_translation) do
+      create(:remote_translation, remote_translatable: proposal, locale: :es)
+    end
+    let(:caller) { RemoteTranslations::Caller.new(remote_translation) }
+
+    it "sanitize field value when the field contains entity references as &Sigma;" do
+      field_values_sanitized = [proposal.title, "Σ with sample text", proposal.summary, proposal.retired_reason]
+      locale = remote_translation.locale
```

Comment:
It looks like this line is not necessary :thinking:.

I highly recommend configuring your editor so it runs `ruby -cw` when you save a Ruby file and tells you about errors and warnings reported by the Ruby interpreter (not to be confused with rubocop, which has many more rules and is much slower). For example, with Vim this is automatically done when installing syntastic.